helm/cloud-provider-cloud-director-app/charts/cloud-provider-for-cloud-director/values.yaml

```diff
@@ -0,0 +1,38 @@
+basicAuthSecret:
+  create: true # Set to false if you create the secret somewhere else
+  username: ""
+  password: ""
+  refreshToken: ""
+
+cpiConfigmap:
+  site: "" # VCD endpoint with the format https://VCD_HOST. No trailing '/'
+  org: "" # VCD organization name where the cluster should be deployed
+  ovdc: "" # VCD virtual datacenter name where the cluster should be deployed
+  ovdcNetwork: "" # VCD virtual datacenter network to be used by the cluster
+  vipSubnet: "" # Virtual IP CIDR for the external network
+  clusterid: "" # "Infra Id" in VCDCluster object in management cluster
+  vAppName: ""
+  enableVirtualServiceSharedIP: false # supported for VCD >= 10.4 - not used yet
+  oneArm:
+    startIP: "192.168.8.2" # Will disappear in future versions
+    endIP: "192.168.8.100" # Will disappear in future versions
+  immutable: true
+
+ccmDeployment:
+  replicas: 1
+  revisionHistoryLimit: 2
+  dnsPolicy: Default
+  imagePullPolicy: IfNotPresent
+  image: projects.registry.vmware.com/vmware-cloud-director/cloud-provider-for-cloud-director

```

Comment:
IMO we should retag it using [our tool](https://github.com/giantswarm/retagger) so we control the precedence and ensure availability